streams/ht.m3u

```diff
@@ -19,7 +19,19 @@ https://watch.haitilive.net/hml/labrisetv/index.m3u8
 https://5790d294af2dc.streamlock.net/8124/8124/playlist.m3u8
 #EXTINF:-1 tvg-id="TVPanou.ht",TV Panou (720p) [Not 24/7]
 http://tvpanoucom.srfms.com:1935/tvpanoucom/livestream/playlist.m3u8
-#EXTINF:-1 tvg-id="",Haiti News [Not 24/7]
+#EXTINF:-1 tvg-id="",Haiti News (720p) [Not 24/7]
 https://haititivi.com/website/haitinews/index.m3u8
 #EXTINF:-1 tvg-id="",Radio Télévision Shilo [Not 24/7]
 https://watch.haitilive.net/freehb/rtvs/index.m3u8
+#EXTINF:-1 tvg-id="NETALKOLETV.ht",NETALKOLE TV (720p)
+https://watch.haitilive.net/naktvhd/index.m3u8
+#EXTINF:-1 tvg-id="TelePacific.ht",Tele Pacific (576p)
+https://pacific.zekletech.com:2002/live/acw_01/index.m3u8
+#EXTINF:-1 tvg-id="TeleBoston.us",Tele Boston (480p)
+http://rtb.srfms.com:3333/live/telebostonrtb/playlist.m3u8
+#EXTINF:-1 tvg-id="NagoTV.us",Nago TV (240p)
+https://lakay.online/public/nagotv/index.fmp4.m3u8
```

Comment:
According to the channel IDs, these two links should be placed in [streams/us.m3u](https://github.com/iptv-org/iptv/blob/master/streams/us.m3u) instead.

Reply:
Done